src/stylesheets/components/_banner.scss

```diff
@@ -1,206 +1,258 @@
   @include typeset($theme-font-family-banner);
-  @include u-bg('base-lightest');
-  @include u-padding-bottom(1);
-  @include u-position('relative');
+  background-color: color('base-lightest');
 
-  @include at-media('mobile-lg') {
-    @include u-font-size($theme-font-family-banner, $theme-font-size-h6);
-    @include u-padding-bottom(0);
+  @include at-media('tablet') {
+    font-size: font-size($theme-font-family-banner, '3xs');
+    padding-bottom: units(0);
   }
 }
 
 .usa-banner-content {
-  @include u-bg('transparent');
-  @include u-padding-bottom(2px);
-  @include u-padding-top(2.5);
-  @include u-padding-x($site-margins-mobile);
-  @include u-width('full');
+  @include grid-container($theme-site-max-width);
+  background-color: color(transparent);
+  font-size: font-size($theme-font-family-banner, 4);
+  overflow: hidden;
+  padding-bottom: units(2);
+  padding-top: units(0.5);
+  padding-right: units($theme-site-margins-mobile);
+  padding-left: units($theme-site-margins-mobile - 1);
+  width: 100%;
 
   @include at-media('tablet') {
-    @include u-padding-bottom(3);
-    @include u-padding-top(5);
+    @include u-padding-y(3);
+  }
+
+  @include at-media('desktop') {
+    @include u-padding-x($theme-site-margins);
   }
 
-  @include at-media($theme-nav-width) {
-    @include u-padding-x($site-margins);
+  p:first-child {
+    margin: 0;
   }
+}
 
-  p {
-    &:first-child {
-      @include u-margin-top(1);
+.usa-banner-guidance-gov {
+  padding-top: units(2);
 
-      @include at-media('tablet') {
-        @include u-margin-top(0);
-      }
-    }
+  @include at-media('tablet') {
+    padding-top: units(0);
+  }
+}
+
+.usa-banner-guidance-ssl {
+  padding-top: units(2);
+
+  @include at-media('tablet') {
+    padding-top: units(0);
   }
 }
 
 .usa-banner-inner {
-  @include grid-container($theme-site-max-width);
-  @include u-padding-x($theme-site-margins-mobile);
+  @include grid-container;
+  @include grid-row;
+  @include u-flex('align-start');
+  padding-left: units($theme-site-margins-mobile);
+  padding-right: units(0);
 
-  @include at-media('mobile-lg') {
-    @include grid-row;
+  @include at-media('tablet') {
     @include u-flex('align-center');
   }
 
-  @include media($nav-width) {
+  @include at-media('desktop') {
     @include u-padding-x($theme-site-margins);
   }
 }
 
 .usa-banner-header {
-  @include u-padding-y(0.5);
-  @include u-font-size($theme-font-family-banner, 1);
+  @include u-padding-y(1);
+  font-size: font-size($theme-font-family-banner, 1);
+  font-weight: font-weight('normal');
+  min-height: units($touch-target-size);
+  position: relative;
 
-  @include at-media('mobile-lg') {
-    @include u-padding-y(2px);
+  @include at-media('tablet') {
+    @include u-padding-y(.5);
+    min-height: 0;
   }
 
-  p {
-    @include u-line-height($theme-font-family-banner, 1);
-    @include u-margin-bottom(0);
-    @include u-margin-top(2px);
-    display: block;
-    overflow: visible;
+  &-close-text {
```

Comment:
Something to consider is this technique it makes it harder for folks to find the selector via `control + f`, since it's not fully there.

Reply:
good point, I'll write it in the standard style